### PR TITLE
Bugfix AB#10610 - Allow Search on Assignee Field

### PR DESCRIPTION
**Pull request by AndreGAot**
- Allow search to pick up the multiple assignees rows by adding a hidden column
- Cleanup some css errors
- There looks like many changes but that is just the formatter applied and adding a column

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css [hunk 1]
```diff
@@ -67,23 +65,22 @@ td.dt-editable:hover {
     color: white;
 }
 
-td.dt-editable:before {
-    content: '\E70F';
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css:71]: removed this icon as default

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css [hunk 2]
```diff
@@ -67,23 +65,22 @@ td.dt-editable:hover {
...
-    font-weight: 100;
-    font-style: normal;
-    font-size: 11px;
-    right: 2px;
-    margin-right: 4px;
-    margin-left: -8px;
-    color: var(--bc-colors-blue-background, #38598A);
-}
+    td.dt-editable:before {
+        font-family: 'Segoe-Fluent-Icons', sans-serif;
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css:69]: added fall back font

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css [hunk 3]
```diff
@@ -67,23 +65,22 @@ td.dt-editable:hover {
...
+        margin-right: 4px;
+        margin-left: -8px;
+        color: var(--bc-colors-blue-background, #38598A);
+    }
 
 .clear-icon:before {
     content: '\E8BB';
     color: var(--bc-colors-white-primary-500, #FFF);
-    font-family: 'Segoe-Fluent-Icons';
+    font-family: 'Segoe-Fluent-Icons', sans-serif;
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css:83]: added fall back comment

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css [hunk 4]
```diff
@@ -102,7 +99,7 @@ td.dt-editable:before {
 
 .select-checkbox {
     border-radius: 8px;
-    color: 1px solid var(--bc-colors-blue-background, #38598A);
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css:105]: fixed wrong css

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js [hunk 5]
```diff
@@ -162,44 +162,44 @@
             cell.setAttribute('contenteditable', true);
             cell.addEventListener('focus', function (e) {
                 checkUserDivChanged();
-                
-                if (e.target.children.length == 0) {
-
+                if (e.target.children.length == 0 ||
+                    (e.target.children.length == 1
+                    && e.target.children[0].className.includes('dt-select-assignees'))) {
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js:167]: update the logic to select the assignees here

**Review by don-aot — APPROVED** [review]

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js [hunk 6]
```diff
@@ -232,198 +232,217 @@
...
+                        render: function (data, type, row) {
+                            let displayText = ' ';
+
+                            if (data != null && data.length == 1) {
+                                displayText = type === 'fullName' ? getNames(data) : data[0].assigneeDisplayName;
+                            } else if (data.length > 1) {
+                                displayText = type === 'fullName' ? getNames(data) : l('Multiple assignees')
+                            }
+
+                            return `<span class="d-flex align-items-center dt-select-assignees">
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js:369]: update this style

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js [hunk 7]
```diff
@@ -232,198 +232,217 @@
...
+                                + getNames(data) + '">' + displayText + '</span>' +
+                                `</span>`;
+                        },
                     },
-                },
-            ],
-        })
-       );
+                    { //9
+                        title: l('Assignee'),
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js:377]: added hidden field for assignees to search on

**Pull request merged**